rumqttd/src/main.rs

```diff
@@ -1,7 +1,12 @@
-use rumqttd::{Broker, Config};
+use std::io::Write;
+
+use config::FileFormat;
+use rumqttd::Broker;
 
 use structopt::StructOpt;
 
+pub static RUMQTTD_DEFAULT_CONFIG: &str = include_str!("../demo.toml");
```

Comment:
used `include_str!` macro instead of copying the config in source code directly was to avoid need to change config file in 2  places whenever we want to update the default config file.

in this case we just need to remember to update this path whenever configuration file name is update or path is changed.